### DISCUSSION TITLE
Start minimized

### DISCUSSION THREAD

**Discussion by Nosnahc** [filips123/PWAsForFirefox#320] [Ideas]
Hello everybody,
I use this extension to easily check but mail on ProtonMail.
I added a shortcut for the PWA to start with my windows (in order to have the notifications) but two things bother me.

There is no "easy" option to add a PWA to windows startup.
There is no option to launch a minimized PWA (and therefore not be in full screen when my OS starts).

So I thought to share an idea for improvement.

In the meantime, thank you very much for your work!

Thank you for your attention.
Regards

**Comment by filips123** [comment]
I plan to implement a few auto launch features, but I cannot promise when. I'll also check if there's any way to enable minimized launch.

**Reply by filips123** [reply]
I've added an option to enable launching at system login/startup. It will be available in the next release. However, I haven't added support for minimized launch, because I don't know yet how to properly implement it.
